config/locales/views/en.yml

```
@@ -2075,6 +2075,46 @@ en:
+        fandom_and_megafandom_dir_html: "%{fandom_and_megafandom_dir_link} - All currently available fandom & megafandom wiki pages"
+        fandom_and_megafandom_dir_link_label: Directory of Fandom & Megafandom Wiki Pages
+        fandoms_in_need_html: "%{fandoms_in_need_link} - Interested in helping out a fandom in need?"
+        fandoms_in_need_link_label: Fandoms in need of Cowranglers
+        otw_slack_chat_html: "%{otw_slack_chat_link} - For more information please see the %{tag_wrangling_slack_tutorial_link}"
+        otw_slack_chat_link_label: OTW Slack Chat
+        tag_wrangler_training_html: "%{tag_wrangler_training_link} - New wranglers, start here!"
+        tag_wrangler_training_link_label: Tag Wrangling Training
+        tag_wranglers_mailing_list_html: "%{tag_wranglers_mailing_list_link} - Message Archive for the Tagwranglers Mailing List"
+        tag_wranglers_mailing_list_link_label: Tag Wranglers Mailing List Conversations
+        tag_wrangling_bugs_html: "%{tag_wrangling_bugs_link} - Having technical issues while wrangling? Check here for solutions!"
+        tag_wrangling_bugs_link_label: Tag Wrangling Bugs & Troubleshooting
+        tag_wrangling_committee_html: "%{tag_wrangling_committee_link} - Full collection of wiki links & policies"
+        tag_wrangling_committee_link_label: Tag Wrangling Committee
+        tag_wrangling_communication_policy_html: "%{tag_wrangling_communication_policy_link} - Includes instructions for leaving tag comments and chat rules"
+        tag_wrangling_communication_policy_link_label: Tag Wrangling Communication Policy
+        tag_wrangling_faq_html: "%{tag_wrangling_faq_link} - Check here to find answers to frequently asked questions"
+        tag_wrangling_faq_link_label: Tag Wrangling FAQ
+        tag_wrangling_moodle_html: "%{tag_wrangling_moodle_link} - Additional modular training and support for all wranglers"
+        tag_wrangling_moodle_link_label: Tag Wrangling Moodle
+        tag_wrangling_slack_tutorial_link_label: Tag Wrangling Slack Tutorial
+        unassigned_fandoms_html: "%{unassigned_fandoms_link} - A list of all current unassigned fandoms"
+        unassigned_fandoms_link_label: Unassigned Fandoms
+        vacations_and_retiring_html: "%{vacations_and_retiring_link} - Unable to wrangle for more than 2 weeks? Read about options for taking a break from wrangling!"
+        vacations_and_retiring_link_label: Vacation and Retiring from Wrangling
+      general_resources_heading: 'General Wrangling Resources:'
+      heading: Tag Wrangling
+      notes:
+        chair_contact_html: For confidential issues (including difficulty reaching other wranglers), please email Chairs at %{tag_wranglers_chair_email} or use the phrase <strong>"Question for a Chair!"</strong> in %{tw_wranglers_slack_link}, %{tw_wranglers_quietroom_slack_link}, or in any training room and let them know you would like to chat via direct message.
```

```suggestion
        chair_contact_html: For confidential issues (including difficulty reaching other wranglers) please email Chairs at %{tag_wranglers_chair_email} or use the phrase "<strong>Question for a Chair!</strong>" in %{tw_wranglers_slack_link}, %{tw_wranglers_quietroom_slack_link}, or in any training room and let them know you would like to chat via direct message.
```
Please match the text given in the Jira issue exactly. The note regarding extracting the strong tag also applies here

If the text in the Jira issue and the HTML in the Jira issue disagree, please pick the HTML version